### PR TITLE
Possible VAMC past events fix

### PR DESCRIPTION
**Pull request by mpelzsherman**
Possible fix for https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6800

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/eventListingPage.graphql.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const eventListingPage = `
         {field: "status", value: "1", operator: EQUAL, enabled: $onlyPublishedContent}, 
         {field: "moderation_state", value: "archived", operator: NOT_EQUAL}, 
         {field: "type", value: "event"},
-        {field: "field_datetime_range_timezone", value: [$today], operator: SMALLER_THAN}]},
+        {field: "field_datetime_range_timezone.end_value", value: [$today], operator: SMALLER_THAN}]},
```
Comment (kelsonic) [inline src/site/stages/build/drupal/graphql/eventListingPage.graphql.js:19]: The code was previously expecting `field_datetime_range_timezone` which is an object, but `field_datetime_range_timezone.end_value` is a number so I think we'll get quite a few runtime errors with this, right? 🙂 I think we might be able to fix it without touching the graphql at all

Comment (kelsonic) [reply src/site/stages/build/drupal/graphql/eventListingPage.graphql.js:19]: Oh wow, jenkins passed though so maybe I'm wrong!

Comment (mpelzsherman) [reply src/site/stages/build/drupal/graphql/eventListingPage.graphql.js:19]: Yep! I confirmed this change in graphql explorer before I pushed up the PR.

Comment (kelsonic) [reply src/site/stages/build/drupal/graphql/eventListingPage.graphql.js:19]: Sweet! Did you test it out locally yet?

Comment (mpelzsherman) [reply src/site/stages/build/drupal/graphql/eventListingPage.graphql.js:19]: Nope. This is unplanned work. But I will circle back and do some more testing later if nobody else is able to.

**Pull request closed without merging**